vllm/config.py

```diff
@@ -376,7 +378,7 @@ def __init__(
         hf_token: Optional[Union[bool, str]] = None,
         hf_overrides: Optional[HfOverrides] = None,
         override_neuron_config: Optional[dict[str, Any]] = None,
-        override_pooler_config: Optional["PoolerConfig"] = None,
+        override_pooler_config: Optional[PoolerConfig] = None,
```

Comment:
It would be better to submit unrelated changes in their own PR to keep review size smaller